gateway-contracts/test/utils/contracts.ts

```diff
@@ -40,73 +40,73 @@ async function initTestingWallets(nKmsNodes: number, nCoprocessors: number, nCus
   await checkIsHardhatSigner(pauser);
 
   // Load the KMS transaction senders
-  const kmsTxSenders = [];
-  for (let idx = 0; idx < nKmsNodes; idx++) {
-    const kmsTxSender = await hre.ethers.getSigner(getRequiredEnvVar(`KMS_TX_SENDER_ADDRESS_${idx}`));
-    await checkIsHardhatSigner(kmsTxSender);
-    kmsTxSenders.push(kmsTxSender);
-  }
+  const kmsTxSenders = await Promise.all(
```

Comment:
Particularly for this utility snippet, which is used only for unit tests, I'd lean toward the current sequential implementation: it's much more readable and debuggable, and the performance difference from `Promise.all` is likely negligible for the number of KMS nodes we typically have.

Wdyt? @jatZama